codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +138,114 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        // TODO: Is this a good size ?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)